### PR TITLE
qemu.tests: Update some cases of migration_multi_host

### PR DESCRIPTION
**Pull request by spcui**
Update cases migration_multi_host_cancel and
migration_multi_host_with_file_transfer.

Signed-off-by: Shuping Cui <scui@redhat.com>

id: 1256111 1256113

### CODE REVIEW

File: qemu/tests/migration_multi_host_cancel.py [hunk 1]
```diff
@@ -51,39 +52,34 @@ def check_guest(self):
...
-                          " cpu and memory load." %
-                          (self.srchost, self.dsthost, mig_protocol),
-                          logging.info)
-            self.migrate_wait(["vm1"], srchost, dsthost, worker)
+                error.context("Load stress in guest before migration.", logging.info)
+                logging.debug("Sending command: %s" % (stress_cmd))
+                session.sendline(stress_cmd)
+
+            cancel_delay = params.get("cancel_delay")
+            error.context("Do migration with migrate_cancel", logging.info)
```
Comment (suqinhuang) [inline qemu/tests/migration_multi_host_cancel.py]: we can remove the line and add it to framework

File: qemu/tests/migration_multi_host_cancel.py [hunk 2]
```diff
@@ -51,39 +52,34 @@ def check_guest(self):
...
-
-            error.context("Migration from %s to %s over protocol %s with high"
-                          " cpu and memory load." %
-                          (self.srchost, self.dsthost, mig_protocol),
-                          logging.info)
-            self.migrate_wait(["vm1"], srchost, dsthost, worker)
+                error.context("Load stress in guest before migration.", logging.info)
+                logging.debug("Sending command: %s" % (stress_cmd))
+                session.sendline(stress_cmd)
+
```
Comment (suqinhuang) [inline qemu/tests/migration_multi_host_cancel.py]: How about move stress test out of worker

File: qemu/tests/migration_multi_host_cancel.py (hunk 1 above)
Comment (spcui) [inline qemu/tests/migration_multi_host_cancel.py]: OK, will update it.

File: qemu/tests/migration_multi_host_cancel.py (hunk 2 above)
Comment (spcui) [inline qemu/tests/migration_multi_host_cancel.py]: Don't need to move it.

**Comment by suqinhuang** [comment]
LGTM

Acked-by: Suqin Huang <shuang@redhat.com>

**Pull request merged**